packages/website/components/Sidebar.tsx

```diff
@@ -72,43 +120,26 @@ export function Sidebar({ currentPage = '/' }: Props) {
             links={sidebarLinks.guidelines}
             currentPage={currentPage}
           />
-
           <SidebarSection
             title="Tokens"
             links={sidebarLinks.tokens}
             currentPage={currentPage}
           />
-
-          <SidebarSection
-            title="Layout Components"
-            links={sidebarLinks.layoutComponents}
-            currentPage={currentPage}
-          />
-
-          <SidebarSection
-            title="Typography Components"
-            links={sidebarLinks.typographyComponents}
-            currentPage={currentPage}
-          />
-
           <SidebarSection
-            title="Form Components"
-            links={sidebarLinks.formComponents}
+            title="Components"
+            links={components}
             currentPage={currentPage}
           />
-
           <SidebarSection
-            title="Components"
-            links={[]}
+            title="Advanced components"
```

Comment:
I am not sure if the Workbench is `Advanced component`. If we want to have this section then maybe we need to move couple more components there, like dunno Card, Modal, etc. WDYT?

Reply:
I think where should all components that are not included in `f36-components`. `Pagination` will be one of the next ones.

Reply:
As soon as we have Pagination component then this makes more sense to me. if this section is only for Workbench then I would suggest not creating it for now. Especially since we are thinking about deprecating Workbench. I don't want to get into a very deep discussion about it now, so it's not a blocker, but I think we are introducing a lot of changes into navigation without validating our assumptions (order or the items, sections).